Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -386,68 +232,100 @@ void AudioPane::OnEmulationStateChanged(bool running)
+                 "select %1.</dolphin_emphasis>");
+  static const char TR_WASAPI_DEVICE_DESCRIPTION[] =
+      QT_TR_NOOP("Selects an output device to use.<br><br><dolphin_emphasis>If unsure, select "
+                 "Default Device.</dolphin_emphasis>");
+  static const char TR_LATENCY_SLIDER_DESCRIPTION[] = QT_TR_NOOP(
+      "Sets the audio latency in milliseconds. Higher values may reduce audio crackling. Certain "
+      "backends only.<br><br><dolphin_emphasis>If unsure, leave this at 20 ms.</dolphin_emphasis>");
+  static const char TR_DOLBY_DESCRIPTION[] =
+      QT_TR_NOOP("Enables Dolby Pro Logic II emulation using 5.1 surround. Certain backends only. "
+                 "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_DOLBY_OPTIONS_DESCRIPTION[] = QT_TR_NOOP(
+      "Adjusts the quality setting of the Dolby Pro Logic II decoder. Higher presets increases "
+      "audio latency.<br><br><dolphin_emphasis>If unsure, select High.</dolphin_emphasis>");
+  static const char TR_STRETCH_ENABLE_DESCRIPTION[] = QT_TR_NOOP(
+      "Enables stretching of the audio to match emulation speed. <br><br><dolphin_emphasis>If "
+      "unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_STRETCH_SLIDER_DESCRIPTION[] = QT_TR_NOOP(
+      "Size of the stretch buffer in milliseconds. Lower values may cause audio "
+      "crackling.<br><br><dolphin_emphasis>If unsure, leave this at 80 ms.</dolphin_emphasis>");
+  static const char TR_VOLUME_DESCRIPTION[] =
+      QT_TR_NOOP("Adjusts audio output volume.<br><br><dolphin_emphasis>If unsure, leave this at "
+                 "100%.</dolphin_emphasis>");
+
+  m_dsp_hle->SetTitle(tr("DSP HLE"));
+  m_dsp_hle->SetDescription(tr(TR_HLE_DESCRIPTION));
+  m_dsp_lle->SetTitle(tr("DSP LLE Recompiler"));
+  m_dsp_lle->SetDescription(tr(TR_LLE_DESCRIPTION));
+  m_dsp_interpreter->SetTitle(tr("DSP LLE Interpreter"));
+  m_dsp_interpreter->SetDescription(tr(TR_INTERPRETER_DESCRIPTION));
+
+  m_backend_combo->SetTitle(tr("Audio Backend"));
+  m_backend_combo->SetDescription(
+      tr(TR_AUDIO_BACKEND_DESCRIPTION)
+          .arg(QString::fromStdString(AudioCommon::GetDefaultSoundBackend())));
+
+  m_dolby_pro_logic->SetTitle(tr("Dolby Pro Logic II Decoder"));
+  m_dolby_pro_logic->SetDescription(tr(TR_DOLBY_DESCRIPTION));
+
+  m_dolby_quality_combo->SetTitle(tr("Decoding Quality"));
+  m_dolby_quality_combo->SetDescription(tr(TR_DOLBY_OPTIONS_DESCRIPTION));
 
-QString AudioPane::GetDPL2ApproximateLatencyLabel(AudioCommon::DPL2Quality value) const
-{
-  switch (value)
+#ifdef _WIN32
+  m_wasapi_device_combo->SetTitle(tr("Output Device"));
+  m_wasapi_device_combo->SetDescription(tr(TR_WASAPI_DEVICE_DESCRIPTION));
+#endif
+
+  m_stretching_enable->SetTitle(tr("Audio Stretching"));
+  m_stretching_enable->SetDescription(tr(TR_STRETCH_SLIDER_DESCRIPTION));
```

Comment:
This one got mixed. Should be `m_stretching_enable->SetDescription(tr(TR_STRETCH_ENABLE_DESCRIPTION));`